synapse/handlers/sliding_sync.py

```diff
@@ -122,6 +134,62 @@ def copy_and_replace(self, **kwds: Any) -> "_RoomMembershipForUser":
         return attr.evolve(self, **kwds)
 
 
+@attr.s(slots=True, frozen=True, auto_attribs=True)
+class _SortedRoomMembershipForUser(_RoomMembershipForUser):
+    """
+    Same as `_RoomMembershipForUser` but with an additional `bump_stamp` attribute.
+
+    Attributes:
+        event_id: The event ID of the membership event
+        event_pos: The stream position of the membership event
+        membership: The membership state of the user in the room
+        sender: The person who sent the membership event
+        newly_joined: Whether the user newly joined the room during the given token
+            range
```

Comment:
```suggestion
    Attributes:
```

I don't think we should double document these attributes?